tests/unit/test_engine_dao.py

```diff
@@ -450,3 +453,62 @@ def test_migration_db_v20(engine_dao):
         for row in rows:
             # The request_uid default value is None
             assert not row[0]
+
+
+def test_db_init_at_v21(tmp_path, engine_dao):
+    """
+    Cover the new migration object code.
+    Compare the new init migration with the old system.
+    """
+    tmp_datatbase = Path(tmp_path / str(uuid4()))
+    try:
+        with sqlite3.connect(tmp_datatbase) as conn:
```

Comment:
You could simply use `:memory:` here, it will save us from having to use a temporary file.

Reply:
The `tmp_datatbase` file is also used by the `engine_dao` fixture for comparison purpose.